### ISSUE TITLE
Clearance on unnecessary redundant semantic elements used and WCAG Failures

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1540]
What to do with unnecessary redundant semantic elements?

When, for instance, the heading element and/or navigation element is used properly for the parts which needs them and thus passing 1.3.1 BUT the heading element is used unnecessary and redundant on many other places on the page, or the nav element is used a lot of times...

This leads to the situation where the correct use is lost due to the "incorrect" use of the same elements for part who do not need it (better not to use it at all!), ALTHOUGH not forbidden in HTML.

This creates an unequal experience BUT no WCAG failure?!

Will this be a gap we need to look at?

**Comment by JAWS-test** [comment]
I think it is a WCAG failure too, see [F43](https://www.w3.org/TR/WCAG20-TECHS/F43.html) and [F46](https://www.w3.org/TR/WCAG20-TECHS/F46.html)

**Comment by mraccess77** [comment]
Use of headings where there should not be is already a failure under WCAG 1.3.1.

**Comment by jake-abma** [comment]
The problem I'm talking about is valid use of html. Like using the nav element 20 times on a page for lists of links,  nested list of links in menus, while html doesn't prescribe this is not allowed.

It's often a debate for a long time what these elements are exactly... but it causes confusion for users... 

Like, who defines what "major navigation" entails, and can I add a nav for every nested ul with links?

**Comment by patrickhlauke** [comment]
yes, i would have thought that marking something up as a heading that is not visually a heading (including empty/whitespace) already counts as a failure of 1.3.1. ditto for something that visually doesn't give the idea of "this is a navigation region/section" but is nonetheless unnecessarily marked as a `<nav>` or similar. there's obviously some gray areas there though, where sometimes markup may go beyond what is visually presented, but for good reason.

**Comment by patrickhlauke** [comment]
> It's often a debate for a long time what these elements are exactly... but it causes confusion for users...
> 
> Like, who defines what "major navigation" entails, and can I add a nav for every nested ul with links?

do you think the working group would be able to define this normatively? I'd have my doubts. the current loose wording gives some subjectivity/wiggle room which i think simply can't be tied down, unless the WG wants to set up a giant list of do's and don't's for structures on all possible web pages on the web...which sounds sisyphean to me. perhaps adding a note to the 1.3.1 understanding could help hint at this, but i can't imagine a hard pass/fail normative way to address this myself...

**Comment by JAWS-test** [comment]
> Like using the nav element 20 times on a page for lists of links, 

I would say: No problem, if they are actually all separate navigation regions with links and the regions are labeled meaningfully

> nested list of links in menus

I would say: Is a problem because the whole menu is a navigation region, but not the individual submenus. It would also be a violation if I marked a list (visually a list) in the source code as many separate lists

**Comment by jake-abma** [comment]
this is an actual site I tested

![20201123_175217](https://user-images.githubusercontent.com/6384002/99991333-30716380-2db5-11eb-90a1-edb71d8bc58b.jpg)

There is no requirement it MUST have a ACC name and the code is valid.

Why would this be a violation?

There is an inequality thought for users and it's frustrating to see we can't fail it.

**Comment by patrickhlauke** [comment]
> There is an inequality thought for users and it's frustrating to see we can't fail it.

that's the fine line between usability and accessibility, i'd say. and the desire to fail because "otherwise clients won't change things". there are likewise many other things that are annoying for other user groups, but technically don't fail WCAG. a page with 150+ individual links, maybe even grouped with properly marked-up headings (so AT users can jump directly to those headings at least) will be a pain for keyboard-only (no AT) users. etc.

**Comment by JAWS-test** [comment]
> There is no requirement the NAV element MUST have a ACC name and the code is valid. Why would this be a violation?

1. https://www.w3.org/TR/wai-aria-practices-1.2/#aria_lh_step3
2. If I have many navigation regions, it is probably clear to a sighted person what they are used for because of their position and visual design. Therefore, according to SC 1.3.1, this should also be transmitted with AT. If there is no visual indication of what they are for, it would not be a violation strictly according to WCAG. In that case, at most a recommendation for labelling could be given with reference to the link under point 1.

**Comment by jake-abma** [comment]
ARIA practices are an advise, not requirement, not a violation

**Comment by jake-abma** [comment]
I'll keep it on the list for future discussion, closing, thx all!
